tests/InputFiles/GeneralizedHarmonic/KerrSchild.yaml

```diff
@@ -33,11 +33,12 @@ AnalyticSolution:
     Center: [0.0, 0.0, 0.0]
 
 EvolutionSystem:
-  GeneralizedHarmonic:
-    Gauge:
-      RollOnStartTime: 100000.0
-      RollOnTimeWindow: 100.0
-      SpatialDecayWidth: 50.0
+  DhGaugeParameters:
+    RollOnStartTime: 100000.0
+    RollOnTimeWindow: 100.0
+    SpatialDecayWidth: 50.0
+    Amplitudes: [1.0, 1.0, 1.0]
```

Comment:
Can you please add a note here saying that these particular choices of amplitude coefficients and exponents come from `SpEC` and should/could be good for binary evolutions as well.

Reply:
... except these parameters won't necessarily work for a binary (you would want 3 gaussians, not one, in that case)

Reply:
@prayush are you ok with the message I added in my fixup commit? If not, please let me know how you'd like me to change it

Reply:
I'm fine with whatever you two decide, so squash without waiting for me :)

Reply:
Looks good to me!